specification/nhais-adaptor.yaml

```diff
@@ -3,7 +3,7 @@
 # owned by NHS Digital (https://digital.nhs.uk/)
 openapi: "3.0.0"
 info:
-  version: "{VERSION}"
+  version: "0.1.0"
   title: NHAIS Adaptor (FHIR) API
```

Comment:
Can we change this to 'GP Links - NHAIS'

Reply:
Did a find/replace across the documentation